controlplane/internal/controller/agentcontrolplane_controller.go

```diff
@@ -93,12 +96,26 @@ func (r *AgentControlPlaneReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		}
 		return ctrl.Result{}, err
 	}
+	log.Info("Started reconciling AgentControlPlane")
```

Comment:
I know we never cared so far, but I'd suggest to set logging right: is this Info or Debug?

Reply:
I was following what assisted-service [does](https://github.com/openshift/assisted-service/blob/66a3ef7405297ff5b3dd21feee1427f03f068dda/internal/controller/controllers/agent_controller.go#L116) but I have no preference, what do you think it should be?

Reply:
Let's agree on some logging "rules":
* when to use what log level (i.e. when to use `INFO`, when to use `DEBUG`, etc)
* what information we should log

We can discuss this offline and come up with a strategy, feel free to ignore this comment for this PR.

Reply:
Discussed with Riccardo offline and determined that 
1. we will review logging in each PR 
2. follow general guidelines:

    - log info when necessary to know and not repetitive
    - log debug levels if it's necessary for debugging
3. A refactor of all logging will be done later on
https://issues.redhat.com/browse/MGMT-17959